inc/TRestRawSignalChannelActivityProcess.h

```diff
@@ -45,34 +45,36 @@ class TRestRawSignalChannelActivityProcess : public TRestEventProcess {
     TRestDetectorReadout* fReadout;  //!
 #endif
 
+    void InitFromConfigFile() override;
+
     void Initialize() override;
 
     void LoadDefaultConfig();
 
    protected:
     /// The value of the lower signal threshold to add it to the histogram
-    Double_t fLowThreshold = 25;
+    Double_t fLowThreshold;
 
     /// The value of the higher signal threshold to add it to the histogram
-    Double_t fHighThreshold = 50;
+    Double_t fHighThreshold;
 
     /// The number of bins at the daq channels histogram
-    Int_t fDaqChannels = 300;
+    Int_t fDaqHistogramChannels;
 
     /// The number of bins at the readout channels histogram
-    Int_t fReadoutChannels = 128;
```

Comment:
It is fine to rename the metadata members althought I would avoid to do that frequently.

But, in order to apply proper class versioning, i.e. ROOT schema evolution, it is required to increase the `ClassDefOverride` number to 4.

I.e. the last line in this file
```
 ClassDefOverride(TRestRawSignalChannelActivityProcess, 3);
```

should be updated to
```
 ClassDefOverride(TRestRawSignalChannelActivityProcess, 4);
```

Reply:
Yes, I think it is better to put the original names again. They are still appropriate names and we would avoid any issues with past versions. Do you agree?

Reply:
Yes, I agree with both choices, still changing names should be not be a big deal as soon as the class version is increased.